README.md

```diff
@@ -0,0 +1,61 @@
+# Mosaic 
+
+> :warning: **This package in under early development and not ready for production use, yet.**
+
+`mosaic` provides the functionality to visualize unstructured mesh data on it's native grid within `matplotlib`. 
+Currently `mosaic` only supports MPAS meshes, but future work will add support for other unstructured meshes used in `E3SM`.
+
+## (Developer) Installation 
+
+Assuming you have a working `conda` installation, you can install the latest development version of `mosaic` by running: 
+```
+conda env create --file dev-environment.yml
+conda activate mosaic-dev
+
+python -m pip install -e .
+```
+
+If you have an existing `conda` environment you'd like install the development version of `mosaic` in, you can run: 
+```
+conda install --file dev-environment.yml
+
+python -m pip install -e .
+```
+
+## Example Usage
+
+First we need to download a valid MPAS mesh. To do so run:
+```
+curl https://web.lcrc.anl.gov/public/e3sm/inputdata/ocn/mpas-o/EC30to60E2r3/mpaso.EC30to60E2r3.230313.nc -o mpaso.EC30to60E2r3.230313.nc
+```
+
+Then we can use `mosaic` to plot on the native mesh using `matplotlib`. For example:
+```python
```

Comment:
I would suggest adding a few more comments to the example about what each part is used for.

Reply:
Good idea! Thanks for the suggestion.